### ISSUE TITLE
Add canvas draw for contact moniker

### ISSUE THREAD

**Issue by okdistribute** [inkandswitch/backchannel#44]
When creating a moniker for a contact, you could draw the name instead of writing it in a text box.

We can use react-canvas-draw for this, here's a codepen example:

https://codesandbox.io/s/react-canvas-draw-example-forked-mzf4z?file=/src/index.js

Depends on #32. This is a stretch goal after we've gotten the PWA working well on mobile, then if we have time this could make sense to experiment with..
